src/claimable_balances/single_claimable_balance_request.rs

```diff
@@ -1,30 +1,43 @@
 use crate::models::*;
 
+
+#[derive(Default, Clone)]
+pub struct ClaimableBalanceId(String);
+#[derive(Default, Clone)]
+pub struct NoClaimableBalanceId;
+
 /// SingleClaimableBalanceRequest is the struct that implements the type for the /claimable_balances endpoint to get a single claimable balance
-/// [More Details](https://laboratory.stellar.org/#explorer?resource=claimable_balances&endpoint=single&network=test "Single Claimable Balance")
-#[derive(Debug)]
-pub struct SingleClaimableBalanceRequest {
-    /// Claimable Balance ID
-    /// [Stellar Documentation](https://developers.stellar.org/api/resources/claimablebalances/single/ "Claimable Balance ID")
-    claimable_balance_id: Option<String>,
+/// [More Details](https://developers.stellar.org/api/horizon/resources/retrieve-a-claimable-balance) "Single Claimable Balance")
+#[derive(Default)]
+pub struct SingleClaimableBalanceRequest<I> {
+    claimable_balance_id: I,
+}
+
+impl SingleClaimableBalanceRequest<NoClaimableBalanceId> {
+    pub fn new() -> Self {
+        SingleClaimableBalanceRequest::default()
+    }
```

Comment:
We should add some documentation. It seems obvious, but that does this actually do?

Reply:
Moved to https://github.com/Baseflow/stellar-rust-sdk/issues/4